jwk/header_test.go

```diff
@@ -13,6 +13,7 @@ func TestHeader(t *testing.T) {
 	values := map[string]interface{}{
 		"kid":     "helloworld01",
 		"kty":     jwa.RSA,
+		"key_ops": []KeyOperation{KeyOpSign},
 		"use":     "sig",
 		"x5t":     "thumbprint",
```

Comment:
https://tools.ietf.org/html/rfc7517#section-4.3
>   The "use" and "key_ops" JWK members SHOULD NOT be used together;

It is `SHOULD NOT` , but was acceptable because it is for the test.